src/SFA.DAS.AssessorService.Web.Staff/Domain/Roles.cs

```diff
@@ -11,15 +11,31 @@ public static class Roles
         public const string AssessmentDeliveryTeam = "EPA"; // AAD
         public const string ProviderRiskAssuranceTeam = "EPR"; // FHA
         public const string RegisterViewOnlyTeam = "EPV";
+        public const string RoatpGatewayTeam = "APR";
         
         public static bool HasValidRole(this ClaimsPrincipal user)
         {
             return user.IsInRole(CertificationTeam)
                    || user.IsInRole(OperationsTeam)
                    || user.IsInRole(AssessmentDeliveryTeam)
                    || user.IsInRole(ProviderRiskAssuranceTeam)
-                   || user.IsInRole(RegisterViewOnlyTeam);
+                   || user.IsInRole(RegisterViewOnlyTeam)
+                   || user.IsInRole(RoatpGatewayTeam);
+        }
+
+        public static bool HasRoatpRoleOnly(this ClaimsPrincipal user)
+        {
+            if (user.IsInRole(RoatpGatewayTeam)
```

Comment:
Just a bit worried that this is fragile to new roles being added.  Not sure if there is a solution though....